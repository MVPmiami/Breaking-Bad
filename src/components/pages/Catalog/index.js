//import styles from "./style.module.scss";
import CardList from "./../../organisms/CardList";

const Catalog = () => {
  return (
    <div>
      <CardList />
    </div>
  );
};

export default Catalog;
