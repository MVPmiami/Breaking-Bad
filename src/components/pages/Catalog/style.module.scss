@import "./../../../assets/styles/style.scss";

.catalogPage {
  display: flex;
  flex-direction: column;
  align-items: center;
  background-color: $rootBcg;
}
