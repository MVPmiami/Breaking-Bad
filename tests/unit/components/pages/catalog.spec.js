import React from "react";
import Catalog from "../../../../src/components/pages/Catalog";

const setUp = (props) => shallow(<Catalog {...props} />);

describe("should render Catalog component", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });

  it("should contain .catalogPage wrapper", () => {
    const wrapper = component.find(".catalogPage");
    expect(wrapper.length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
