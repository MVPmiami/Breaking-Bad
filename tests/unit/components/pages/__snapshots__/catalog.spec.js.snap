// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Catalog component should contain .catalogPage wrapper 1`] = `
<main
  className="catalogPage"
>
  <main>
    <PersonsListContainer />
  </main>
</main>
`;
